src/internal/sftpgo/sftpgo.go

```diff
@@ -175,20 +157,12 @@ func (a *API) _do(c *http.Client, req *http.Request) ([]byte, error) {
 	return data, err
 }
 
-// GetToken procures a time-sensitive authorization token from SFTPGo.  This
-// doesn't need to be called directly, but can be used as a "health check".
-func (a *API) GetToken() error {
-	a.m.Lock()
-	defer a.m.Unlock()
-
-	if a.token.ExpiresAt.Sub(a.now()) > (3 * time.Minute) {
-		return nil
-	}
-
-	var data, err = a.rpc("GET", "token", "")
+// GetStatus requests SFTPgo server status to verify API key use is successful
+func (a *API) GetStatus() error {
+	var _, err = a.rpc("GET", "status", "")
 	if err != nil {
-		return fmt.Errorf("unable to retrieve token: %w", err)
+		return fmt.Errorf("Unable to retrieve server status: %w", err)
 	}
 
-	return json.Unmarshal(data, a.token)
+	return nil
```

Comment:
Is there any value in checking the response here? Or is it just a simple situation where the http response (e.g., 200) tells us everything we need?

Reply:
This GetStatus() function is called in a couple places and the return value does matter. If there is something wrong it will return an error. If there isn't, `nil` is expected to communicate no error:
- https://github.com/uoregon-libraries/newspaper-curation-app/pull/204/files#diff-06f125df4e056dafea341b849353ead0b4cdf44740af8712b5125844d28b9718R47
- https://github.com/uoregon-libraries/newspaper-curation-app/pull/204/files#diff-03fd1e829e53dbbbd12be3f88ac1df4298bbac18a8372368673d825efcd11a23R67

Is this approach okay?

Reply:
Very poorly worded question on my part, and attached to the wrong line of code. Yeesh. Resolved this in slack.